### ISSUE TITLE
Export folder test not working

### ISSUE THREAD

**Issue by tomchadwin** [tomchadwin/qgis2web#255]
The Export folder test is not working (https://github.com/tomchadwin/qgis2web/blob/master/test/test_qgis2web_dialog.py#L2000):

```python
self.dialog.items['Data export'].get('Export folder').setText(1, '/tmp/customfolder')
```

This fails to set the folder. How do I do it?

**Comment by tomchadwin** [comment]
I've tried various commands, but I'm still not managing to change the text value of a QTreeWidgetItem (second column) in the context of this test. Can anyone help? @walkermatt, @akbargumbira?

**Comment by tomchadwin** [comment]
I think the issue was that I was trying to retrieve the `itemWidget` from that `QTreeWidgetItem`. Because it's text, it doesn't have a widget. I think.

**Comment by tomchadwin** [comment]
All working now. Released as 1.7.0 and pushed to QGIS plugins repo.
